front_end/src/app/app.component.scss

```diff
@@ -1 +1,268 @@
 @import '../assets/scss/main.scss';
```

Comment:
this should probably be it's own file that is imported into here so as not to pollute the main app.scss